src/redux/settings.js

```diff
@@ -1,18 +1,17 @@
-import { NativeCurrencyKeys } from '../entities/nativeCurrencyTypes';
+import { updateLanguage } from '../languages';
+import { NativeCurrencyKeys } from '@rainbow-me/entities';
 import {
   getNativeCurrency,
   getNetwork,
   saveLanguage,
   saveNativeCurrency,
   saveNetwork,
-} from '../handlers/localstorage/globalSettings';
-import { web3SetHttpProvider } from '../handlers/web3';
-import networkTypes from '../helpers/networkTypes';
-import { updateLanguage } from '../languages';
-
-import { ethereumUtils } from '../utils';
-import { dataResetState } from './data';
-import { explorerClearState, explorerInit } from './explorer';
+} from '@rainbow-me/handlers/localstorage/globalSettings';
+import { web3SetHttpProvider } from '@rainbow-me/handlers/web3';
+import networkTypes from '@rainbow-me/helpers/networkTypes';
+import { dataResetState } from '@rainbow-me/redux/data';
+import { explorerClearState, explorerInit } from '@rainbow-me/redux/explorer';
+import { ethereumUtils } from '@rainbow-me/utils';
```

Comment:
While here, I also updated the imports of this file to use the `@rainbow-me` format where possible.